catalyst_voices/lib/pages/registration/show_upload_confirmation_dialog.dart

```diff
@@ -0,0 +1,135 @@
+import 'dart:convert';
+
+import 'package:catalyst_voices/widgets/avatars/voices_avatar.dart';
+import 'package:catalyst_voices/widgets/buttons/voices_filled_button.dart';
+import 'package:catalyst_voices/widgets/buttons/voices_text_button.dart';
+import 'package:catalyst_voices/widgets/modals/voices_alert_dialog.dart';
+import 'package:catalyst_voices/widgets/modals/voices_dialog.dart';
+import 'package:catalyst_voices/widgets/modals/voices_upload_file_dialog.dart';
+import 'package:catalyst_voices_assets/catalyst_voices_assets.dart';
+import 'package:catalyst_voices_brands/catalyst_voices_brands.dart';
+import 'package:catalyst_voices_localization/catalyst_voices_localization.dart';
+import 'package:catalyst_voices_models/catalyst_voices_models.dart';
+import 'package:collection/collection.dart';
+import 'package:flutter/material.dart';
+
+Future<void> showUploadConfirmationDialog(
+  BuildContext rootContext, {
+  ValueChanged<List<SeedPhraseWord>>? onUploadSuccessful,
+  bool Function(List<SeedPhraseWord> value)? onValidate,
+}) async {
+  await VoicesDialog.show<void>(
```

Comment:
We're using more static functions for dialogs. See `AccountPage._removeKeychain` and convert to this style pls